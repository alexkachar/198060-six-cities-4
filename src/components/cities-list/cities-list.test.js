import * as React from 'react';
import renderer from 'react-test-renderer';
import CitiesList from './cities-list';

const CITIES = [
  `Amsterdam`,
  `Brusseles`,
  `Berlin`,
  `Cologne`
];

const selectedCity = `Amsterdam`;

it(`CitiesList renders correctly`, () => {
  const tree = renderer
    .create(<CitiesList
      cities={CITIES}
      selectedCity={selectedCity}
    />, {
      createNodeMock: () => document.createElement(`div`)
    })
    .toJSON();
  expect(tree).toMatchSnapshot();
});
