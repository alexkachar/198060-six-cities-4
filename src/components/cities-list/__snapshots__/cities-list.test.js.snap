// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CitiesList renders correctly 1`] = `
<section
  className="locations container"
>
  <ul
    className="locations__list tabs__list"
  >
    <li
      className="locations__item"
    >
      <a
        className="locations__item-link tabs__item tabs__item--active"
        href="#"
        onClick={[Function]}
      >
        <span>
          Amsterdam
        </span>
      </a>
    </li>
    <li
      className="locations__item"
    >
      <a
        className="locations__item-link tabs__item null"
        href="#"
        onClick={[Function]}
      >
        <span>
          Brusseles
        </span>
      </a>
    </li>
    <li
      className="locations__item"
    >
      <a
        className="locations__item-link tabs__item null"
        href="#"
        onClick={[Function]}
      >
        <span>
          Berlin
        </span>
      </a>
    </li>
    <li
      className="locations__item"
    >
      <a
        className="locations__item-link tabs__item null"
        href="#"
        onClick={[Function]}
      >
        <span>
          Cologne
        </span>
      </a>
    </li>
  </ul>
</section>
`;
